quipucords/api/source/view.py

```diff
@@ -246,3 +177,53 @@ def destroy(self, request, pk):
 
         except Source.DoesNotExist as exception:
             raise Http404 from exception
+
+    @action(detail=False, methods=["post"], url_path="bulk_delete")
+    def bulk_delete(self, request):
```

Comment:
```suggestion
    @action(detail=False, methods=["post"])
    def bulk_delete(self, request):
```
Minor: no need to specify url_path if the method name matches the url you want to use.